2_process/src/process_fcn.R

```diff
@@ -25,9 +23,7 @@ process <- function(input_data, output_file, col_model_type, pch_model_type){
       model_type == 'pgdl' ~ pch_model_type[3]
     ), n_prof = as.numeric(str_extract(exper_id, '[0-9]+')))
   
-  # Save the processed data
-  readr::write_csv(eval_data, file = file.path(output_file))
-  
+
```

Comment:
I would recommend either deleting `eval_data <-` so that the output is directly returned from the function or adding `return(eval_data)` to be very explicit!